Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN export YARN_CACHE_FOLDER="$(mktemp -d)" \
 COPY . .
 RUN yarn build
 
-FROM node:12-alpine
+FROM node:12-alpine as app
```

Comment:
This may not be needed. I'm using `--target app` because I only want to tag this image and not the builder.